metadata-service/factories/src/main/java/com/linkedin/metadata/boot/steps/IngestDataPlatformInstancesStep.java

```diff
@@ -61,7 +61,7 @@ public void execute(@Nonnull OperationContext systemOperationContext) throws Exc
     long numEntities = _migrationsDao.countEntities();
     int start = 0;
 
-    while (start < numEntities) {
+    while (start < (int) numEntities) {
```

Comment:
Shouldn't we change the `start` type to long instead and fix all the relevant issues?

Reply:
`start` is used in listAllUrns below, which expects int

Reply:
And we can't fix `listAllUrns`?